release: v1.4.9 (#2365)
rero/rero-ils: RELEASE-NOTES.rst

Release notes
=============

v1.4.9
------

> This patch brings many improvements and fixes on the notification
> process, such as:

Reviewer: ```suggestion
This patch brings several improvements and fixes on the notification
process:
```

> -  Preventing to send (again and again) notifications that are in an

Reviewer: ```suggestion
-  Preventing to repeatedly send notifications that are in an
```

   error state.

> -  Preventing notification to be in an error state if the pickup

Reviewer: ```suggestion
-  Preventing notifications to be in an error state if the pickup
```

   location is deleted.
-  Cancelling generated notifications if the situation changed between

>    the generation and the processing. In deed, some notifications are

Reviewer: ```suggestion
   the generation and the processing. Indeed, some notifications are
```

   asynchronous in order to be able to group several notifications of

>    the same type to the same patron. But in that delay, the notification

Reviewer: ```suggestion
   the same type to the same patron. But in the meantime, a notification
```

   may be outdated, because the loan has been renewed or checked in.
-  Preventing to send reminders if the library is closed between the due
   date and the notification processing (`#2303`_).
-  Computing the date at which the due soon notification has to be sent when

>    creating the loan, instead of through a recurring task, in order to

Reviewer: ```suggestion
   creating the loan, instead of through a recurring task, in order to improve
```

>    performance and get a more stable date.

Reviewer: ```suggestion
   performance and apply a more stable date.
```

> -  Processing the due soon and overdue notification as soon as possible, to

Reviewer: ```suggestion
-  Processing due soon and overdue notifications as soon as possible, to
```
